webapp/portlet/src/main/webapp/html/spaceSettings.html

```diff
@@ -0,0 +1,12 @@
+<div class="VuetifyApp">
+    <div data-app="true"
+         class="v-application transparent v-application--is-ltr theme--light singlePageApplication"
+         id="SpaceSettings" flat="">
+        <v-cacheable-dom-app cache-id="SpaceSettings"></v-cacheable-dom-app>
```

Comment:
```suggestion
```
No need to use `v-cacheable-dom-app` anymore since this mechanism has evolved and disabled by default.

Reply:
Done !